flaskr/templates/auth/login.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Log In</title>
+    <link rel="stylesheet" href="./../../static/style.css">
+</head>
+<body id="login-body">
+<div class="login-background-container container centered-container vertical-container">
+    <div class="login-header centered-container vertical-container">
+        <img class="login-logo" src="./../images/polar-bear.png" alt="AVA Logo">
+        <div class="login-description">Log In To AVA</div>
+    </div>
+    <div class="login-form-container centered-container vertical-container">
+        <form class="login-form" id="loginForm" action="#" method="post">
+            <input type="email" placeholder="Email Address" name="email">
+            <input type="password" placeholder="Password" name="password">
+        </form>
+        <button class="button sign-in-button" type="submit" form="loginForm">Log In</button>
+        <a class="forgot-password-link" href="">Forgot Password?</a>
+    </div>
+    <div class="divider">or</div>
+    <button class="button sign-up-button" type="button">Create New Account</button>
+</div>        
+</body>
+</html>
+
+ 
```

Comment:
Usually only have 1 new line at the End of File. It's okay no need to update now :)